src/lib.rs

```diff
@@ -39,6 +39,7 @@ use core::convert::TryInto;
 /// Decodes KTX2 texture data
 pub struct Reader<Data: AsRef<[u8]>> {
     input: Data,
+    header: Header,
```

Comment:
Mixed feelings about cacheing this when it's ~free to re-decode on demand, but realistically it doesn't matter and it's an impl detail anyway.

Reply:
I just cached it because it felt natural and because I wanted to avoid an unwrap in the header method.

Reply:
Yeah, I absolutely see where you're coming from there. We can swap it out later if we want, so it's fine.